src/base.ts

```diff
@@ -1,5 +1,5 @@
 import createDebug from 'debug';
-import EventEmitter from 'eventemitter3';
+import {EventEmitter} from 'eventemitter3';
```

Comment:
<img width="528" alt="Screenshot 2023-06-26 at 22 40 50" src="https://github.com/obs-websocket-community-projects/obs-websocket-js/assets/28832095/fcbf6003-9ad4-45d2-98f6-5f5b3dbcdee1">

eventemitter3 exports `EventEmitter` with both named and root exports, but it confuses TypeScript because the default export is a namespace mixed with classes.